### PR TITLE
Add SFM token price on BNB

### PR DESCRIPTION
**Pull request by antonio-mendes**
Hey @RockandGrohl ,

I ended reopening a PR with the required changes. You can see the changes I've applied, basically you just had 2 small things to change, 1 missing come, and having the BNB token in the BNB token area.

More importantly, I had to open a new PR because you actually created forked our repository. This means that you created a copy of our `Spellbook` repository and then did this PR to this copy of our repo. That means your change even if merged wouldn't be seen on Dune because your copy of the repo is not the real one.

I understand this might be a bit confusing if you're starting out so I suggest you try some tutorials or read up on documentation so you start understanding Git better. This is a fun course: https://learngitbranching.js.org/

Thanks for your fist contribution!

### CODE REVIEW

**Comment by RockandGrohl** [comment]
Thanks Antonio, really struggled with this, it seemed like I was on the right path but just pressed the wrong things. Is the token price now live for query on Dune? Thanks
________________________________
From: António Mendes ***@***.***>
Sent: 29 August 2022 23:36
To: duneanalytics/spellbook ***@***.***>
Cc: RockandGrohl ***@***.***>; Mention ***@***.***>
Subject: [duneanalytics/spellbook] Add SFM token price on BNB (PR #1459)


Hey @RockandGrohl<https://github.com/RockandGrohl> ,

I ended reopening a PR with the required changes. You can see the changes I've applied, basically you just had 2 small things to change, 1 missing come, and having the BNB token in the BNB token area.

More importantly, I had to open a new PR because you actually created forked our repository. This means that you created a copy of our Spellbook repository and then did this PR to this copy of our repo. That means your change even if merged wouldn't be seen on Dune because your copy of the repo is not the real one.

I understand this might be a bit confusing if you're starting out so I suggest you try some tutorials or read up on documentation so you start understanding Git better. This is a fun course: https://learngitbranching.js.org/

Thanks for your fist contribution!

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/duneanalytics/spellbook/pull/1459

Commit Summary

  *   d10ef97<https://github.com/duneanalytics/spellbook/pull/1459/commits/d10ef9709efecfe036c5550e139989fb29f2b7db> Add SFM token price on BNB

File Changes

(1 file<https://github.com/duneanalytics/spellbook/pull/1459/files>)

  *   M models/prices/prices_tokens.sql<https://github.com/duneanalytics/spellbook/pull/1459/files#diff-0350e0e6bf4dcce94ae74401c31cb3fe946eaaef8f2b2faa63ad2f4c32d6dbb7> (2)

Patch Links:

  *   https://github.com/duneanalytics/spellbook/pull/1459.patch
  *   https://github.com/duneanalytics/spellbook/pull/1459.diff

**Review by aalan3 — APPROVED** [review]

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -170,7 +170,7 @@ VALUES
     ("tbc-terablock", "bnb", "TBC", "0x9798df2f5d213a872c787bd03b2b91f54d0d04a1", 18),
     ("sps-splintershards", "bnb", "SPS", "0x1633b7157e7638c4d6593436111bf125ee74703f", 18),
     ("dec-dark-energy-crystals", "bnb", "DEC", "0xe9d7023f2132d55cbd4ee1f78273cb7a3e74f10a", 3),
-
+    ("sfm-safemoon", "bnb", "SFM", "0x42981d0bfbaf196529376ee702f2a9eb9092fcb5", 9)
```
Comment (jeff-dude) [inline models/prices/prices_tokens.sql:173]: @aalan3 is this missing a comma?

Comment (antonio-mendes) [reply models/prices/prices_tokens.sql:173]: Ah yeah I think so

Comment (antonio-mendes) [reply models/prices/prices_tokens.sql:173]: Should I do a new PR just adding the comma?

Comment (aalan3) [reply models/prices/prices_tokens.sql:173]: yeah my bad!

**Pull request merged**